### PR TITLE
Add NOTTOOLONG macro for more clear code.

### PR DESCRIPTION
**Pull request by richsalz**
Also fix one missing use of it. Thanks to GitHub user Vort for finding
it and pointing out the fix.

Fied #4085

### CODE REVIEW

File: crypto/rand/rand_win.c [hunk 1]
```diff
@@ -496,11 +495,11 @@ int RAND_poll(void)
                             do
                                 RAND_add(&hentry, hentry.dwSize, 5);
                             while (heap_next(&hentry)
+                                   && NOTTOOLONG(starttime)
```
Comment (Vort) [inline crypto/rand/rand_win.c]: I guess, `!good` check should go here also.

**Comment by richsalz** [comment]
updated commit pushed with extra !good ness :)

File: crypto/rand/rand_win.c [hunk 2]
```diff
@@ -496,11 +495,11 @@ int RAND_poll(void)
                             do
                                 RAND_add(&hentry, hentry.dwSize, 5);
                             while (heap_next(&hentry)
+                                   && (!good || NOTTOOLONG(starttime))
                                    && --entrycnt > 0);
                         }
                     } while (heaplist_next(handle, &hlist)
-                             && (!good
-                                 || (GetTickCount() - starttime) < MAXDELAY));
+                             && (!good || NOTTOOLONG(starttime));
```
Comment (Vort) [inline crypto/rand/rand_win.c]: Looks like this code needs more `)`.
Maybe at other locations too.

**Comment by richsalz** [comment]
can you tell i didn't compile on windows? :)  fixed three places.

**Comment by Vort** [comment]
Looks fine now.

**Comment by richsalz** [comment]
merged, thanks!

**Pull request closed without merging**